src/app/zap-templates/templates/app/attributes/Accessors.zapt

```diff
@@ -14,6 +14,12 @@
 
 namespace chip {
 namespace app {
+
+enum class MarkAttributeDirty {
+    IfChanged,
+    No,
```

Comment:
This enum should be in af-types I think.

Also: `kIfChanged` and `kNo` if possible. A bit uglier, but consistent style.

Reply:
Yep, will do.